scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -62,6 +88,34 @@ trait AndroidAppModule extends JavaModule {
    */
   def androidReleaseKeyPath: T[PathRef] = Task.Source(millSourcePath / androidReleaseKeyName())
 
+  /**
+   * Specifies the file format(s) of the lint report. Available file formats are defined in AndroidLintReportFormat,
+   * such as AndroidLintReportFormat.Html, AndroidLintReportFormat.Xml, AndroidLintReportFormat.Txt, and AndroidLintReportFormat.Sarif.
+   */
+  def androidLintReportFmt: T[Seq[AndroidLintReportFormat.Value]] =
```

Comment:
I'm not sure if `fmt` is used in this codebase as a word, it seems it is rather `format` which is widely used.